File: PYTHON/Kode/Store.py
class Store:
    def __init__(self):
        self.inventory = []
    
    def add_clothing(self, clothing):
        self.inventory.append(clothing)
    
    def display_inventory(self):
        print("+--------------------------------------------------------------+")
        print(f"| {'Brand':<15} | {'Ukuran':<10} | {'Harga':<13} | {'Tipe':<13} |")
        print("+--------------------------------------------------------------+")
        for item in self.inventory:
            print(item.display())
        print("+--------------------------------------------------------------+")

File: C++/Kode/Store.cpp
// Store.h
#ifndef STORE_H
#define STORE_H

#include <vector>
#include "Clothing.cpp"

class Store {
private:
    vector<Clothing*> inventory;

public:
    void addClothing(Clothing* clothing) {
        inventory.push_back(clothing);
    }

    void displayInventory() {
        cout << "+----------------------------------------------------------------+\n";
        cout << "| " << left << setw(15) << "Brand" << " | " << setw(10) << "Ukuran" << " | " << setw(12) << "Harga" << "  | " << setw(15) << "Tipe" << " |" << endl;
        cout << "+----------------------------------------------------------------+\n";
        for (Clothing* c : inventory) {
            c->display();
        }
        cout << "+----------------------------------------------------------------+\n";
    }
};

#endif